### PR TITLE
Bindings for types/wlr_switch.h

### PR DESCRIPTION
**Pull request by pmkap**

### CODE REVIEW

**Review by novakne — COMMENTED** [review]
LGTM other than this small missing field

File: src/types/switch.zig [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+    const Impl = opaque {};
+
+    impl: *const Impl,
+
+    events: extern struct {
+        toggle: wl.Signal(*event.Toggle),
+    },
+    
+    data: usize
```
Comment (ifreund) [inline src/types/switch.zig:35]: Missing a trailing comma here, could you run `zig fmt` to see if it catches anything else?
```suggestion
    data: usize,
```

Comment (pmkap) [reply src/types/switch.zig]: Yes indeed it looks like zig fmt changed a whitespace line.

**Review by ifreund — APPROVED** [review]
LGTM, Thanks!

**Pull request merged**